File: rest/src/main/java/io/sparkled/viewmodel/playlist/PlaylistViewModelConverter.java
package io.sparkled.viewmodel.playlist;

import io.sparkled.model.entity.Playlist;
import io.sparkled.viewmodel.ModelConverter;
import io.sparkled.viewmodel.ViewModelConverter;

public abstract class PlaylistViewModelConverter implements ModelConverter<Playlist, PlaylistViewModel>,
        ViewModelConverter<PlaylistViewModel, Playlist> {
}

File: rest/src/main/kotlin/io/sparkled/viewmodel/playlist/PlaylistViewModelConverter.kt
package io.sparkled.viewmodel.playlist

import io.sparkled.model.entity.Playlist
import io.sparkled.viewmodel.ModelConverter
import io.sparkled.viewmodel.ViewModelConverter

abstract class PlaylistViewModelConverter : ModelConverter<Playlist, PlaylistViewModel>, ViewModelConverter<PlaylistViewModel, Playlist>
